packages/grafana-toolkit/src/config/webpack.plugin.config.ts

```diff
@@ -99,6 +99,7 @@ const getCommonPlugins = (options: WebpackConfigurationOptions) => {
         // If src/README.md exists use it; otherwise the root README
         { from: hasREADME ? 'README.md' : '../README.md', to: '.', force: true },
         { from: 'plugin.json', to: '.' },
+        { from: '**/README.md', to: '[path]README.md' },
```

Comment:
Do we need to make use of [`priority`](https://webpack.js.org/plugins/copy-webpack-plugin/#priority) so we can control how it plays with the `README.md` copy happening above on line 100?

Reply:
@jackw good point https://github.com/grafana/grafana/pull/44068/commits/7d0b2208fbba429779ee10e86f7af1f91e0f1e50